.vib/thanos/bucketweb/cypress/cypress/integration/thanos_bucketweb_spec.js

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
+
+it('allows to see runtime & build information', () => {
+    cy.visit('/');
+    cy.contains('.dropdown-toggle', 'Status').click();
+    cy.contains('[class="dropdown-item"]', 'Runtime & Build Information').click();
+    cy.contains('Runtime Information').should('be.visible');
+    cy.get('h2').contains('Build Information').should('be.visible');
+})
```

Comment:
You are checking whether the headers appear, but not asserting that the actual information appears (e.g. the actual version is **displayed**). We have faced some issues in the past in which the information was missing and a user filed a new issue to let us know.

Maybe it is worth it adding it. That does not imply that you need to check the version matches (you can do that in GOSS), but at least that some info is rendered and it meets certain regex.

Reply:
I've applied additional assertions for the described cases.